docs/abandoned-projects.rst

```diff
@@ -0,0 +1,111 @@
+Abstract
```

Comment:
This should have a descriptive header for our TOC's (eg. "Policy for Abandoned Projects")

Reply:
I've fixed the header and made a few other tweaks to make it work in the context of a larger document.